smif/data_layer/__init__.py

```diff
@@ -470,6 +519,24 @@ def read_scenario_set(self, scenario_set_name):
 
         return filtered_scenario_data
 
+    def read_scenario_config(self, scenario_name):
```

Comment:
The name is not consistent with the other methods
I would either call this 'read_scenario' or update the existing functions to

read_scenario_sets(self) -> read_scenario_config_sets(self)
read_scenario_set(self, scenario_set_name) -> read_scenario_config_set(self, scenario_config_set_name)
write_scenario_set(self, scenario_set): -> ..
update_scenario_set(self, scenario_set): -> ..
write_scenario(self, scenario): -> write_scenario_config(self, scenario):
update_scenario(self, scenario): -> ..

Reply:
In fact I would rather use definition instead as config, to stay in a similar style with region_definition and interval_definition